buildSrc/src/main/kotlin/me/vkryl/task/TaskFunctions.kt

```diff
@@ -213,4 +210,10 @@ fun String.unwrapDoubleQuotes(): String {
   if (!this.startsWith("\"") || !this.endsWith("\""))
     error("Not wrapped: \"${this}\"")
   return this.substring(1, this.length - 1).replace("\\\"", "\"")
+}
+
+fun String.compatRelativePath(): String {
```

Comment:
Instead of calling this function while passing arguments on caller's side in every call, isn't it better to place a single call inside of `writeToFile` function?

Reply:
Yes, if we only expect relative paths. I added the `isRelativePath` parameter to explicitly specify the type of path (by default, `isRelativePath` = true)